docs/skills/ujust-tui.md

```
@@ -0,0 +1,330 @@
+
+## Section separators
+
+Inside `gum choose`, use `──` (U+2500) separators to group items. They must be
+non-actionable — filter them out after selection:
+
+```bash
+[[ "${CHOSEN}" == "──"* || -z "${CHOSEN}" ]] && exit 0
+```
+
+Section separator format — aim for ~53 characters total:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)